examples/ephemeral-spark/tamr-config.tf

```diff
@@ -64,7 +64,7 @@ module "tamr-config" {
   emr_managed_core_sg_id      = module.ephemeral-spark-sgs.emr_managed_sg_id
   # emr_managed_core_sg_id   = "" # you may leave this blank and AWS creates one automatically
   emr_additional_core_sg_id = join(",", module.aws-emr-sg-master.security_group_ids)
```

Comment:
This is the security group for the EMR HBase cluster, which won't include the spark ports. Isn't that an issue?

Reply:
All our tests were successful ran. The Ephemeral Spark cluster spun up, ran the tasks, and destroyed. The configured service access security group allows ingress traffic from the EMR and Spark security groups on port 9443 and the outbound to the same EMR/Spark security groups to port 8443.